app/models/manageiq/providers/openstack/cloud_manager.rb

```diff
@@ -23,6 +23,7 @@ class ManageIQ::Providers::Openstack::CloudManager < EmsCloud
 
   include ManageIQ::Providers::Openstack::ManagerMixin
   include HasManyCloudNetworksMixin
+  include ManageIQ::Providers::Openstack::CloudTenantMixin
```

Comment:
I'm thinking this should be a top-level thing...I can't see why any Cloud Provider that providers tenants wouldn't want this as a platform feature.